corgi/core/models.py

```diff
@@ -808,7 +808,7 @@ def get_purl(self):
                 type="container",
                 namespace=self.Namespace.REDHAT.lower(),
                 name=self.name,
-                version=f"{self.version}",
+                version=f"{self.version}-{self.release}",
```

Comment:
I think we should only add '-' if release is not None or empty.

Reply:
Not related to this PR, but NVR, and NEVRA field for container container components has the same training '-' problem that this would create.

Reply:
There don't seem to be any:
```
In [3]: Component.objects.filter(type=Component.Type.CONTAINER_IMAGE, release="").count()
Out[3]: 0
```
NVR and NEVRA do have the problem for other component types though, but I guess I'll leave that for a follow-up PR.